c/parallel/src/jit_templates/CMakeLists.txt

```
@@ -5,28 +5,62 @@ set(cpp_entry "${CMAKE_CURRENT_LIST_DIR}/jit_entry.h")
 
 file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
 
+# The pre-processor (PP) command differs between MSVC and others.
+set(PP_CMD ${CMAKE_CXX_COMPILER})
+set(PP_DEPFILE_KW)
+set(PP_ARGS)
+
+if(MSVC)
+  list(APPEND PP_ARGS /D_CCCL_C_PARALLEL_JIT_TEMPLATES_PREPROCESS)
+else()
+  list(APPEND PP_ARGS -D_CCCL_C_PARALLEL_JIT_TEMPLATES_PREPROCESS)
+endif()
+
+if(MSVC)
+  file(TO_NATIVE_PATH "${jit_template}" JIT_TEMPLATE_NATIVE)
+
+  list(APPEND PP_ARGS
+    /nologo
+    /TP
+    /P
+    /Fi${JIT_TEMPLATE_NATIVE}
```

question: do you need quotes to make spaces in the path work? 
```suggestion
    /Fi"${JIT_TEMPLATE_NATIVE}"
```